pkg/provider/provider_acceptance_test.go

```diff
@@ -119,6 +121,37 @@ func TestAcc_Provider_configHierarchy(t *testing.T) {
 	})
 }
 
+func TestAcc_Provider_(t *testing.T) {
```

Comment:
what does this test prove? I think there's a missing test title (function name) and I don't get what it's testing. Could you briefly describe?

Reply:
I forgot to add the title 🤦 

It tests what we discussed on Slack, so the situation when setting the client once is ignored for one test and potentially it can leave the client for the next tests in a bad shape.

Reply:
Added helping comments